doc/source/train/deepspeed.rst

```diff
@@ -0,0 +1,34 @@
+.. _train-deepspeed:
+
+Training with DeepSpeed
+=======================
+
+Ray :class:`~ray.train.torch.TorchTrainer` can help you easily distribute your DeepSpeed training over a Ray Cluster.
+Simply put all your existing training logics into a training function, then launch a TorchTrainer.
+
+Below is a simple example of ZeRO-3 training with DeepSpeed only. 
```

Comment:
Can you show an outline of the code before this full e2e example, similar to the Accelerate?